kudos/views.py

```diff
@@ -5,3 +5,8 @@
 @app.route("/")
 def hello():
     return "Hello World!"
+
+
+@app.route("/createUser")
```

Comment:
Leave this change out of this pull request because 1) it doesn't do anything 2) it's unrelated to the feature in this PR.